RIN.Core/Models/ClientApi/ArmyRank.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace RIN.Core.ClientApi
+{
+    public class ArmyRank
+    {
+        public uint      id             { get; set; }
+        public ulong     army_guid      { get; set; }
+        public string    name           { get; set; }
+        public bool      is_commander   { get; set; }
+        public bool      can_invite     { get; set; }
+        public bool      can_kick       { get; set; }
+        public DateTime  created_at     { get; set; }
+        public DateTime? updated_at     { get; set; }
+        public bool      can_edit       { get; set; }
+        public bool      can_promote    { get; set; }
+        public uint      position       { get; set; }
```

Comment:
Position is uint here but in for ArmyMember and AddArmyRankReq rank_position is int, should they not match?